lib/core/profiling.js

```diff
@@ -1,72 +0,0 @@
-/* @flow */
-
-import { MODULE_PROFILING_KEY, PERF_PROFILING_KEY } from '../constants/sessionStorage';
-import { Alert } from '../utils';
-import {
-	headReady,
-	bodyStart,
-	bodyReady,
-	contentLoaded,
-	loadOptions,
-	beforeLoad,
-	go,
-	afterLoad,
-} from './init';
-
-export function startProfiling() {
-	if (sessionStorage.getItem(PERF_PROFILING_KEY)) setupPerf();
-	if (sessionStorage.getItem(MODULE_PROFILING_KEY)) setupModulePerf();
-}
-
-function setupPerf() {
-	const end = {};
-
-	const promises = {
-		headReady,
-		bodyStart,
-		bodyReady,
-		contentLoaded,
-		loadOptions,
-		beforeLoad,
-		go,
-	};
-
-	for (const [key, promise] of Object.entries(promises)) {
-		promise.then(() => { end[key] = performance.now(); });
-	}
-
-	function diff(a, b) {
-		const time = (end[b] - end[a]) | 0;
-		return `<span style="color: ${time < 0 ? 'green' : 'red'}">${time}</span>ms`;
-	}
-
-	afterLoad.then(() => {
-		Alert.open([
-			`beforeLoad stalled for ${diff('headReady', 'loadOptions')}`,
-			`go stalled for ${diff('bodyReady', 'beforeLoad')}`,
-			`afterLoad stalled for ${diff('loadComplete', 'go')}`,
-			`addModuleBodyClasses was late by ${diff('bodyStart', 'loadOptions')}`,
```

Comment:
This kind of profiling was misleading, since it was more dependent on how the browser decided to schedule extensions than actually how fast or how soon they ran.